CDP4IMEInstaller/BasicRDLPlugin.wxs

```diff
@@ -1,17 +1,16 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
 
   <?include variables.wxi ?>
 
 	<Fragment>
     <ComponentGroup Id="BASICRDL_CG" Directory="BASICRDL">
 
-      <Component Id="comp_BasicRdl" Guid="{DE37D6B6-D60F-4B4C-B8A5-537A9FCBDF02}" Location="local" Win64="$(var.Win64)">
```

Comment:
can you please explain why the win64 attribute is no longer required?

Reply:
Because with WiX4, it automatically detect the target architecture. Moreover, it can create 64bits or 32bits MSI, depending

Reply:
https://docs.firegiant.com/wix/fourthree/faqs/#converting-the-componentwin64-attribute For more information